src/index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly avoid disabling linters as this makes your code clean, it would be better to remove this line and make sure that the linters are passing 👍🏿